R/print_contrib_affil.R

```diff
@@ -41,32 +41,75 @@ print_contrib_affil <- function(infosheet) {
                                                     is.na(affiliation) ~ NA_integer_))
   
   # Modify data for printing contributor information
-  contrib_print <- 
+  contrib_data <- 
     contrib_affil_data %>% 
     dplyr::select(-affiliation) %>% 
-    tidyr::spread(key = affiliation_type, value = affiliation_no) %>% 
-    dplyr::transmute(contrib = purrr::pmap_chr(list(Names, `Primary affiliation`, `Secondary affiliation`),
-                                               ~ dplyr::if_else(is.na(..3),
-                                                                glue::glue("{..1}^{..2}^"),
-                                                                glue::glue("{..1}^{..2},{..3}^")))) %>% 
-    dplyr::pull(contrib) %>% 
-    glue::glue_collapse(., sep = ", ")
+    tidyr::spread(key = affiliation_type, value = affiliation_no)
+  
+  if (output_format == "rmd") {
+    contrib_print <-
+      contrib_data %>% 
+      dplyr::transmute(contrib = purrr::pmap_chr(list(Names, `Primary affiliation`, `Secondary affiliation`),
+                                                 ~ dplyr::if_else(is.na(..3),
+                                                                  glue::glue("{..1}^{..2}^"),
+                                                                  glue::glue("{..1}^{..2},{..3}^")))) %>% 
+      dplyr::pull(contrib) %>% 
+      glue::glue_collapse(., sep = ", ")
```

Comment:
To reduce redundancies a little these two lines could be applied after the if-else-block, no?